frontend/app/src/components/home/Home.svelte

```diff
@@ -697,6 +697,14 @@
         });
     }
 
+    function freezeGroup(ev: CustomEvent<string>) {
```

Comment:
There's no need to bubble this stuff all the way up to Home now that we have global access to the client. We should just handle it directly in the MessageEntry component.